components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -195,6 +195,24 @@ struct LeoModelOptions {
 
 struct CustomModelOptions {
   string model_request_name;
+
+  // This value is provided by the user when adding/configuring a custom model.
+  // If not provided, Brave will default to using
+  // ai_chat::kCustomModelDefaultContextSize.
+  uint32? context_size;
```

Comment:
These optional ints are causing the iOS build to fail (there's a thread in slack as to why). Whilst we can ignore structs from the iOS build, I don't think we can ignore CustomModelOptions as it's part of the type that's including on Model, which iOS needs. Not sure iOS has implemented a settings dialog for it yet though, so maybe we can replace it with a more basic type, but it could get complicated.
One way to get around this is an optional struct, e.g. `CustomContextSize? context_size` and put the non-optional `uint32` values on that type that.

Reply:
I was just about to work on a fix for this after reviewing those iOS logs. I think we can make these required, like they are for leo models. After all, with some of the previous commits in place, the values are always provided. Earlier, they weren't. So I should be able to make this change.

Reply:
Just pushed a commit which should resolve the iOS build(s), making these members required.